docs/UserGuide.md

```diff
@@ -221,7 +236,7 @@ Example:
 #### 3.3.4. Record leave taken by staff: `addl`
 
 Records leave taken by a staff that is in Eva. <br>
-Format: `addleave INDEX l/d/DATE [d/DATE]`
+Format: `addleave INDEX l/d/DATE [d/DATE] [l/d/DATE [d/DATE]]…​`
```

Comment:
is this addl instead?

Reply:
right lol

Reply:
i got a bit more commits to push anyways wait ah